hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java

```diff
@@ -679,6 +679,10 @@ public OmKeyInfo lookupKey(OmKeyArgs args, String clientAddress)
       throw new OMException("Key not found", KEY_NOT_FOUND);
     }
 
+    if (!args.getFullLocationVersion()) {
```

Comment:
This flag is set in lookupFile, getFileStatus, readKey/readFile in RpcClient but on Server, I don't see any use of that.

Reply:
Since OM will always get the full location from rocksdb, the flag here is for OM to decide whether to send the full version back to client.

Reply:
Got it.